server/migrations/27_add_test_runner.down.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+DROP TABLE "test_runners";
```

Comment:
I think the table could be just `test_runner`, what do you think?

Reply:
All of the tables have plural names so I think I'm going to stick with that